### PR TITLE
Unit model

### PR DESCRIPTION
**Pull request by REYNALopez12**
The units table model was created

### CODE REVIEW

File: app/models/service.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Service < ApplicationRecord
+    has_many : trajectory
```
Comment (jesusMava) [inline app/models/service.rb]: why this have the same changes?

File: app/models/unit.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class Unit < ApplicationRecord
+    has_one : trajectory
+    has_many : type
```
Comment (mtopete1996) [inline app/models/unit.rb]:
```suggestion
    has_one :trajectory
    has_many :types
```

File: lib/trajectories/route_10.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (mtopete1996) [inline lib/trajectories/route_10.json]: No

File: lib/trajectories/route_21.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
+{
```
Comment (mtopete1996) [inline lib/trajectories/route_21.json]: No

**Review by mtopete1996 — CHANGES_REQUESTED** [review]

**Review by mtopete1996 — APPROVED** [review]

**Pull request merged**